### ISSUE TITLE
gce module not support new zone in asia.

### ISSUE THREAD

**Issue by ingramchen** [ansible/ansible#7270]
```
  tasks:

    - name: Launch instances
      gce:
          machine_type: "{{ machine_type }}"
          image: "{{ image }}"
          zone: "asia-east1-b"
      register: gce
```

both new zone `asia-east1-a` and `asia-east1-b` are not supported.

**Comment by jimi-c** [comment]
This appears to potentially be an issue with the way libcloud is working under the hood. CC'ing @erjohnso, as he's written most of the gce module stuff.

**Comment by erjohnso** [comment]
That's likely my bad in ansible vs libcloud, but I'll take a look and see what I can do to fix it up.

**Comment by jimi-c** [comment]
I was looking through both our code, but didn't see anywhere we were hard-coding the list of zones. I also checked libcloud's GCE driver code and didn't see anything there, so I'm not quite sure why this would be failing.

**Comment by ingramchen** [comment]
I am not sure, is it because hard coding choices ?

https://github.com/ansible/ansible/blob/devel/library/cloud/gce#L358

```
zone = dict(choices=['us-central1-a', 'us-central1-b',
           'us-central2-a', 'europe-west1-a', 'europe-west1-b'],
            default='us-central1-a'),
```

**Comment by jimi-c** [comment]
@ingramchen yes, after discussing it with @erjohnso we spotted the issue and I will have a fix merged shortly. Thanks!

**Comment by jimi-c** [comment]
The above patch should resolve this issue, and will be included in 1.6. Please let us know if you continue having problems. Thanks!

**Comment by erjohnso** [comment]
Thanks for reporting this @ingramchen and thanks for the quick fix @jimi-c !!
